python/py-pytest-subtests/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        pytest-dev pytest-subtests 0.4.0
+name                py-${github.project}
+revision            0
+categories-append   devel
+platforms           darwin
+supported_archs     noarch
+license             MIT
+fetch.type          git
+
+python.versions     37 38 39
```

Comment:
this also needs a PY36 subport since it's needed for `py36-cryptography`, see the `lint` error: " Error: Unknown dependency: py36-pytest-subtests"

Reply:
Thanks done.